aggregator/config.go

```diff
@@ -60,7 +60,7 @@ type Config struct {
 	ChainID uint64
 
 	// ForkID is the L2 ForkID provided by the Network Config
-	ForkId uint64 `mapstructure:"ForkId"`
+	ForkId uint64
```

Comment:
How do you assign this variable? If it is not needed you can delete it

Reply:
It is assigned in run.go, after read the forkid intervals from L1 (ChanID is also assigned in this way):
https://github.com/0xPolygonHermez/zkevm-node/blob/b801f527572adb919ebf3d99be8bc603ddc24a70/cmd/run.go#L132